### DISCUSSION TITLE
Team Introductions

### DISCUSSION THREAD

**Discussion by MauriceReeves-usds** [CDCgov/prime-reportstream#1196] [General]
Hey ReportStream team members, please take a moment to introduce yourself below, what your area of responsibility is on the team, what's your area of expertise, and how people can get in touch with you.

**Comment by MauriceReeves-usds** [comment]
My name is Maurice Reeves, I'm an engineer with USDS working on the ReportStream project. I'm primarily working on onboarding and maintaining the connections with the states and the counties, which means that I not only build the engineering logic around connecting to states, but I'm also involved in doing customer service with them, answering emails, investing any problems if they arise, etc.

I currently live near Harrisburg, PA.

The best way to reach me is to ping me on here, or on Slack, or send me an email: qva8@cdc.gov

**Comment by RickHawesUSDS** [comment]
Hi, my name is Rick Hawes. I'm the project lead for ReportStream and one of the engineers on the project. I was also part of the original PRIME sprint team. I've contributed to most of the ReportStream's workstreams. 

Five facts about me:
- I live in Googleville CA which is in the Pacific time zone
- I try to keep Eastern time zone hours, getting into work around 9:30 ET
- I used to work at the Centers for Medicare and Medicaid on the Blue Button project
- Before joining USDS, I worked a couple of very large tech firms, the last one being Yahoo, where I was a VP of engineering. 
- I'm a big fan of the local sports teams, having season tickets to the Warriors and the 49ers. 

The best way to reach me is Slack or rhawes@cdc.gov.

**Comment by ronaldheft-gov** [comment]
Hi, my name is Ron Heft. I’m the DevOps Lead for the PRIME project, which includes both ReportStream and SimpleReport.

Within ReportStream I manage the Azure environment, maintain our Terraform configuration, ensure our CI/CD pipeline runs smoothly, and work with the CDC on certifying the security of our systems.

I live outside of DC in Montgomery Village, MD and maintain EST hours. I am typically available over Slack between 10am - 5pm EST. The best way to contact me is Slack or emailing me at qwp3@cdc.gov.

**Comment by carlosfelix2** [comment]
Greetings and salutations!  My name is Carlos Felix and I am a Senior Software Engineer with [Agile 6](agile6.com) working on Report Stream.   
I live in Rockledge, FL, close to Cape Canaveral and on EST time.  Best way to contact me is Slack and optionally carlos.felix@agile6.com

**Comment by tlosbo** [comment]
My name is Tom Osborne, I'm an engineer with DataRobot working on the ReportStream project. I am currently working on bug fixes and enhancements to support ReportStream features.

I currently live in Louisville, KY.

The best way to reach me is to ping me on here, Slack, or email: [tom.osborne@datarobot ](mailto:tom.osborne@datarobot.com)

**Comment by mry-usds** [comment]
Hi, my name is Matt Young and I'm a USDS engineer working on the ReportStream project.  Primarily, I work the front end, but still operate as a full-stack engineer playing around with the APIs and backend.

I live in Tucson, AZ which puts me in my own little time zone since we don't observe DST - right now I'm on Pacific time - but will switch back to Mountain time during the winter.

You can best reach me on Slack or via email at qtv1@cdc.gov.

**Comment by td-usds** [comment]
Hi, my name is Thomas. I’m working on the DevOps side of things for the PRIME project, which includes both ReportStream and SimpleReport.

I'm based out of DC and maintain ET hours (but we all know UTC is the one that really matters :) ). Slack is the best way to get a hold of me. You can also e-mail me at <not_yet_provisioned>@cdc.gov (no e-mail yet since I'm currently still in the process of getting onboarded).

**Comment by drew-usds** [comment]
Hey I'm Drew and I forgot to do this last month! I'm co-leading design across the ReportStream teams with @celestemespinoza, and am focusing on UI/UX and any/all user-facing aspects of the project.

I'm in DC and keep ET hours, but _does anybody really know what time it is_?

Reach me on Slack or by email at `qop4 ((at)) cdc.gov`.

**Comment by nicholeweems** [comment]
Hello ReportStream team! I am Nichole and a Portfolio Director with Fearless. My role is to support the badgeless team as needed. Prior to my current role, I have a background in accounting, product management and scrum.

Feel free to contact me via Slack @ Nichole Weems, nweems@fearless.tech or (443)604-1032 during ET hours. Feel free to reach out to me at anytime. I also may reach out at any time, yet will indicate FUTURE ___ to clearly communicate there is no expectation that you will respond immediately.

**Comment by oifekoya** [comment]
Hello ReportStream! I am Yemi a Scrum Master at Fearless. My role is to help support and provide guidance to the team, to deliver quality product and services. You can reach me via slack @ Opeyemi Ifekoya , email - oifekoya@fearless.tech or 2404690108 during ET hours. So excited to be on this project!

**Comment by derekkinsman** [comment]
Hey ReportStream, I'm Derek. I'm a developer at Coforma and am supporting the Fearless team on the project. Excited to be helping out here!

I'm based in Guelph, Canada, can be reached on here, Slack, [email](derek@coforma.io), or phone (416-371-5652). I'm in Eastern Time but am happy to do any time shifting if it helps with folx in other timezones.

**Comment by inkandsound** [comment]
Hi everyone! I'm Dawn. I'm a UX designer supporting the Fearless team on this project via A1M Solutions. I have a background in front-end dev (happiest sticking to CSS/HTML though), marketing, and writing/editing. Excited to be joining the team!

I'm based in Chicago (Central Time) - Slack is the best way to get me, but you can also email dawn.low@a1msolutions.com. Feel free to ping me later in the day as well.
